model/event/subscriptions.go

```diff
@@ -456,6 +456,7 @@ func NewPatchOutcomeSubscriptionByOwner(owner string, sub Subscriber) Subscripti
 	return NewSubscriptionByOwner(owner, sub, ResourceTypePatch, triggerOutcome)
 }
 
+// TODO PM-794: also create one of these for requester = trigger?
```

Comment:
PM tickets aren't public, just create the ticket and avoid the comment.

Reply:
oops I had already opened EVG-5399 but forgot to remove this